### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by tianpf**

### CODE REVIEW

**Review by zhyt1985 — APPROVED** [review]

File: algorithm/calc/parameter.go [hunk 1]
```diff
@@ -107,10 +107,10 @@ const (
...
 	// FALSE Dimension value of the empty geometry (-1).
 	// TRUE Dimension value of non-empty geometries (= {P, L, A}).
-	// DONTCARE Dimension value for any dimension (= {FALSE, TRUE}).
-	FALSE    = -1
-	TRUE     = -2
-	DONTCARE = -3
+	// DONOTCARE Dimension value for any dimension (= {FALSE, TRUE}).
+	FALSE     = -1
+	TRUE      = -2
+	DONOTCARE = -3
```
Comment (xiaogu-space) [inline algorithm/calc/parameter.go]: "DONOT_CARE" is better ?

Comment (tianpf) [reply algorithm/calc/parameter.go]: Because I want to use camel case code style，const variable should be defined uniformly

Comment (tianpf) [reply algorithm/calc/parameter.go]: take and done

Comment (xiaogu-space) [reply algorithm/calc/parameter.go]: OK

**Review by xiaogu-space — APPROVED** [review]

**Pull request merged**